payjoin/src/receive/mod.rs

```diff
@@ -873,6 +894,7 @@ mod test {
         use bitcoin::{Address, Network};
 
         let proposal = proposal_from_test_vector().unwrap();
+        assert_eq!(proposal.psbt_fee_rate().unwrap().to_sat_per_vb_floor(), 2);
```

Comment:
I understand why you might make psbt_fee_rate public for this function, but it's so brief and nonessential to the payjoin api I'd rather see it repeated in the test than exposed publically.

Reply:
the pub was a leftover from when i was testing in integration, removed that. 
here in the unit test it can access all the functions/structs in the same module so its fine.